src/token.rs

```diff
@@ -30,30 +33,55 @@ impl From<jsonwebtoken::errors::Error> for TokenError {
 
 #[derive(Debug, Serialize, Deserialize, PartialEq)]
 pub struct Claims {
-    pub sub: String,
-    pub company: String,
+    pub user: String,
+    pub scope: String,
+    pub client_id: String,
     pub iat: i64,
     pub exp: i64,
 }
 
+fn read_from_file(path: &str) -> Vec<u8> {
+    std::fs::read(path).unwrap_or_else(|error| {
```

Comment:
FYI: for one time reads like this, or for what are basically statics, it might be useful to use https://doc.rust-lang.org/std/macro.include_str.html. Not important here, but just so you know.

Reply:
Thanks for the reference, I will keep that in mind.